composer.json

```diff
@@ -32,12 +46,12 @@
         "phpunit/php-token-stream": "^3.0.1",
         "phpunit/php-text-template": "^1.2.1",
         "sebastian/code-unit-reverse-lookup": "^1.0.1",
-        "sebastian/environment": "^4.0",
+        "sebastian/environment": "dev-pcov",
```

Comment:
You can require `dev-pcov as 4.0.99` to alias that package here and same below for phpunit I suppose.

Reply:
Ah I see you just removed the aliases. I don't know why they didn't work.